src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CreditWaiter.cs

```diff
@@ -51,11 +51,10 @@ public void ResetForAwait(CancellationToken cancellationToken)
         private void RegisterCancellation(CancellationToken cancellationToken)
         {
             _cancellationToken = cancellationToken;
```

Comment:
We can get rid of _cancellationToken now, right?

Reply:
It's used below for a CanBeCanceled check. Potentially a refactoring could get rid of that, but I didn't go looking at all of the callers of that helper.